packages/frontend/app/components/school-session-type-form.hbs

```diff
@@ -2,13 +2,10 @@
   <div
     class="school-session-type-form"
     data-test-school-session-type-form
-    {{did-insert (perform this.load)}}
```

Comment:
something broken with this refactor. clicking the "save" button on this form does nothing now now - regardless of updating or adding an item.